STX.SPAL.Core.Tests.Unit/STX.SPAL.Core.Tests.Unit.csproj

```diff
@@ -6,6 +6,7 @@
     <Nullable>disable</Nullable>
     <IsPackable>false</IsPackable>
     <IsTestProject>true</IsTestProject>
+    <WarningLevel>1998</WarningLevel>
```

Comment:
Should this be `<NoWarn>CS1998</NoWarn>`

Reply:
it doesn´t work, that´s the reason I wrote just the number.

![image](https://github.com/user-attachments/assets/6393ac59-0839-40e7-b3ba-0f95f2646796)

Reply:
Oh!! my bad, it´s NoWarn